src/spootnik/reporter/impl.clj

```diff
@@ -523,7 +523,7 @@
   (capture! [this e]
     (capture! this e {}))
   (capture! [_this e tags]
-    (rs/send-event! (:dsn sentry) sentry-options e tags))
+    (rs/send-event! sentry sentry-options e tags))
```

Comment:
I'm not sure I got that change, are you sure it's not a breaking one?

Reply:
It's breaking because the namespace is public, but consumers should not be using `rs/send-event!` anyway.